bin/test-run

```diff
@@ -35,6 +35,30 @@ function run_test(){
     go test -v "$1" --linkerd="$linkerd_path" --linkerd-namespace="$linkerd_namespace" --k8s-context="$k8s_context" --integration-tests "$2"
 }
 
+function install_edge() {
+  set -x
+  tmp=$(mktemp -d -t l5dbin.XXX)
+  trap "rm -rf $tmp" EXIT
+
+  curl https://run.linkerd.io/install-edge | HOME=$tmp sh
+
+  local linkerd_path=$tmp/.linkerd2/bin/linkerd
+  $linkerd_path install --proxy-auto-inject --linkerd-namespace="$linkerd_namespace" | kubectl apply -f - > /dev/null 2>&1
```

Comment:
i think we want to skip `--proxy-auto-inject` in the initial edge installation, and then introduce the auto injector when we do the upgrade. related, in `install_test.go`, this would also decouple the `--proxy-auto-inject` from the `TestHelper.UpgradeFrom()` setting, so the command a few lines below this becomes:

```bash
run_test "$test_directory/install_test.go" "--proxy-auto-inject" "--upgrade-from-version=$edge_version" "--linkerd-namespace=$linkerd_namespace" || exit_code=$?
```